onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -74,6 +76,83 @@ static Env& platform_env = Env::Default();
 #pragma warning(push)
 #endif
 
+using PyCallback = std::function<void(std::vector<py::object>, py::object user_data, std::string)>;
```

Comment:
Is passing vector<py::object> by value intentional?

Reply:
it is intentional yes.